### DISCUSSION TITLE
App support for Switchbot outdoor temp/humidity (W340001X) ?

### DISCUSSION THREAD

**Discussion by jebissell** [theengs/app#101] [Ideas]
My W340001X was automatically detected on my Home Assistant server and also displays in the Switchbot app on Android 14. I want to carry the sensor with me when traveling outside the home, and would therefore like to use Theengs to connect via MQTT. Will app support for this sensor arrive soon, seeing that other Switchbot sensors are supported already?

As a minor point, it might be helpful to add a "supported in app" column to the compatibility list, to avoid the confusion I had when purchasing the app, thinking my sensor was supported.

**Comment by DigiH** [comment]
As the actual decoder for the SwitchBot Outdoor Meter should already be included in the current App version I have to leave it to the team members being involved in the actual app development itself to see if there might be a compatibility issue with the Outdoor Meter due to its dual servicedata **and** manufacturerdata requirement - something unique to the Outdoor Meters, compared to other SwitchBot devices - or something else.

> As a minor point, it might be helpful to add a "supported in app" column to the compatibility list, to avoid the confusion I had when purchasing the app, thinking my sensor was supported.

But this already does exist, with the column named **Mobile App**

https://decoder.theengs.io/devices/devices.html

**Reply by jebissell** [reply]
My further error, it wasn't obvious in my browser that there was a further column.

Fingers crossed the team can get that sensor working, since Theengs seems to be the only solution for my use case.

**Comment by Olivetti** [comment]
Count me in. I also bought the app, not seeing the lacking support for the W340001X.
Hope they get it working.
